mesos-swarm-marathon/configure-mesos-cluster.sh

```diff
@@ -63,22 +112,63 @@ if isagent ; then
   echo "this node is an agent"
 fi
 
-zkconfig()
+zkhosts()
 {
-  postfix="$1"
-  zkconfigstr="zk://"
+  zkhosts=""
   for i in `seq 1 $MASTERCOUNT` ;
   do
     if [ "$i" -gt "1" ]
     then
-      zkconfigstr="${zkconfigstr},"
+      zkhosts="${zkhosts},"
     fi
-    zkconfigstr="${zkconfigstr}${MASTERPREFIX}${i}:2181"
+    zkhosts="${zkhosts}${MASTERPREFIX}${i}:2181"
   done
-  zkconfigstr="${zkconfigstr}/${postfix}"
+  echo $zkhosts
+}
+
+zkconfig()
+{
+  postfix="$1"
+  zkhosts=$(zkhosts)
+  zkconfigstr="zk://${zkhosts}/${postfix}"
   echo $zkconfigstr
 }
 
+################
+# Install Docker
+################
+
+echo "Installing and configuring docker and swarm"
+
+time wget -qO- https://get.docker.com | sh
+
+# Start Docker and listen on :2375 (no auth, but in vnet)
+echo 'DOCKER_OPTS="-H unix:///var/run/docker.sock -H 0.0.0.0:2375"' | sudo tee /etc/default/docker
+sudo service docker restart
+
+ensureDocker()
+{
+  # ensure that docker is healthy
+  dockerHealthy=1
+  for i in {1..3}; do
+    sudo docker run hello-world
```

Comment:
You are pulling image down to user's machine and not deleting it. You're not deleting the container earlier. 

I suggest just running "docker info" @anhowe